lesson_10/Lesson10_Task32_met.java

```diff
@@ -0,0 +1,66 @@
+                "голову, уставился на колбу и вдруг  схватил  ее.  Пыли  на  ней  не  стало\n" +
+                "меньше, выцветший ярлычок нисколько не изменился, но Хэллем тем  не  менее\n" +
+                "крикнул:\n" +
+                "     - Черт подери! Какой сукин сын трогал эту колбу?\n" +
+                "     Так по крайней мере утверждал Денисон, который слышал  этот  вопль  и\n" +
+                "много  лет  спустя  поведал   о   нем   Ламонту.   Парадный   рассказ   об\n" +
+                "обстоятельствах замечательного открытия, запечатленный во множестве книг и\n" +
+                "учебников,  этой  фразы  не  содержит.  Перед  читателем  возникает  образ\n" +
+                "проницательного химика, который орлиным взором сразу же подметил изменения\n" +
+                "и мгновенно сделал далеко идущие выводы.\n" +
+                "     Куда там! Хэллему  вольфрам  был  не  нужен,  он  его  совершенно  не\n" +
+                "интересовал. И, в сущности, ему было все равно, трогал  кто-то  колбу  или\n" +
+                "нет. Просто он (подобно многим другим людям) терпеть не мог, когда на  его\n" +
+                "столе  хозяйничали  без  его  ведома,  и  всегда  готов  был   заподозрить\n" +
+                "окружающих в таких посягательствах, продиктованных исключительно  желанием\n" +
+                "ему насолить.\n" +
+                "     Но в покушении на колбу никто не признавался. Бенджамин Аллан Денисон\n" +
+                "услышал возглас Хэллема потому, что сидел  в  кабинете  напротив  лицом  к\n" +
+                "открытой двери. Он поднял голову и встретил сверлящий взгляд Хэллема.\n" +
+                "     Хэллем не внушал ему  особых  симпатий  (впрочем,  он  никому  их  не\n" +
+                "внушал), а в  то  утро  Денисон  плохо  выспался  и  -  как  он  вспоминал\n" +
+                "впоследствии - был даже рад сорвать на ком-нибудь свое дурное  настроение.\n" +
+                "Хэллем же был для этого идеальным объектом.";
+
+        StringTokenizer st = new StringTokenizer(text, ".,!?:;()\"- \n");                   // create StringTokenizer object and define all delimiters
+        List<String> list = new ArrayList<>();                                                      // create new ArrayList to store delimited elements of our text
+        Map<String, Integer> freqVocab = new TreeMap<>();                                       // create new TreeMap to store our frequency vocabulary
+        while(st.hasMoreTokens()) {                                                             // delimit our text to words using lower case and save them to ArrayList
+            list.add(st.nextToken().toLowerCase());
+        }
+
+        for (int i = 0; i < list.size(); i++) {                                                 // go through all the words in ArrayList, count repeatings and save the result to our frequency vocabulary
```

Comment:
It's too expensive operation - for every word start two cycles. It's better to do it with only one cycle and using HashMap put on every word occurrence (just check if word already in map - add 1 to value, if not - create new key-value)